backend/app/controllers/spree/admin/product_properties_controller.rb

```diff
@@ -8,6 +8,16 @@ class ProductPropertiesController < ResourceController
       before_action :setup_property, only: :index, if: -> { can?(:create, model_class) }
       before_action :setup_variant_property_rules, only: :index
 
+      def update_positions
+        params[:positions].each do |id, index|
+          Spree::ProductProperty.find(id).set_list_position(index)
```

Comment:
Isn't this raising an exception calling `set_list_position` on `nil` if the `find(id)` can't find the record?

Reply:
You are right, this record is not found. 
But the resource controller catches the exception:
rescue_from ActiveRecord::RecordNotFound, with: :resource_not_found
But this doesn't hurt because the redirect of the json request to resource_not_found does nothing.

This is also why the update_position method in the resource controller didn't work and needed to be overriden. The difference is that the resource controller does the position updates in a Transaction and then does a rollback -> so no sorting happens!

i would say this is not the best solution to override the action.

so i did update this PR to do a cleanup of the update positions as a before_action.
Is that ok? Or is there a better solution?